bcgov/HOUS-permit-portal
# [bphh-856] Bugfix with scheduling template versions and template version formatting

## Description
Explicitly set formatting and date calculations for template version date to use Vancouver timezone, as otherwise there are bugs with scheduling



## What type of PR is this? (check all applicable)

- [X ] 🐛 Bug Fix

## Related Tickets & Documents
https://hous-bssb.atlassian.net/browse/BPHH-856


## Steps to QA
It is difficult to recreate the bug with scheduling, however we can check for the stability of the version date
- Ensure you have either a scheduled or published template
- Change browser timezone to different timezones. No matter the timezone the version date should remain the same, especially if the chosen timezone has an offset causing the current browser time to be in the next day e.g. (Mumbai time)
